### ISSUE TITLE
future.magic template repository

### ISSUE THREAD

**Issue by krlmlr** [HenrikBengtsson/future.tests#21]
similar to https://github.com/r-dbi/RKazam/ .

Would you mind to relicense https://github.com/HenrikBengtsson/future/blob/develop/R/UniprocessFuture-class.R under CC0 or something similar so that we can license the template package liberally?

**Comment by HenrikBengtsson** [comment]
Please expand (much more). ELI5

**Comment by krlmlr** [comment]
I'm proposing to create a template repository that implements the simplest possible future that passes all future.tests. In addition to https://github.com/HenrikBengtsson/future/blob/develop/R/UniprocessFuture-class.R?rgh-link-date=2022-05-05T10%3A00%3A08Z we might need code from a few other files. Would you be open to supporting such a template repository under a permissive license, perhaps CC0?
